libs/Wallet/qml/Status/Wallet/AssetsPanel.qml

```diff
@@ -0,0 +1,167 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import Status.Wallet
+
+import Status.Onboarding
+
+import Status.Containers
+
+Item {
+    id: root
+
+    /// WalletController
+    required property WalletController controller
+
+    ColumnLayout {
+        anchors.left: leftLine.right
+        anchors.top: parent.top
+        anchors.right: rightLine.left
+        anchors.bottom: parent.bottom
```

Comment:
Temporary until we port the needed `StatusQ` components to `Qt6`. Otherwise here I anchor the content between the left and right lines. Therefore content left binds to the right of the left line and content right to the left of the right line. More time and attention would have been made this simpler :)